app/src/main/java/com/duckduckgo/app/browser/omnibar/OmnibarScrolling.kt

```diff
@@ -25,17 +25,24 @@ import javax.inject.Inject
 
 class OmnibarScrolling @Inject constructor() {
 
+
+    private var isScrollingEnabled = true
+
+    val isOmnibarScrollingEnabled:Boolean get() = isScrollingEnabled
```

Comment:
I would expect that I could safely call any of the methods in this class (`enableOmnibarScrolling`, `disableOmnibarScrolling`) without needing to check  `isOmnibarScrollingEnabled`. But not sure about this.

If so, I don't think we need this property here and we will avoid needing to update the value all the time.